src/controllers/assistant.js

```diff
@@ -98,20 +102,21 @@ function AssistantController(context) {
     const { data: { command, prompt, options }, attributes } = requestContext;
     const { firefallArgs = {} } = commandConfig[command] || {};
     const { imageUrl, ...otherOptions } = options || {};
-    const { authInfo: { profile } } = attributes;
+    const { authInfo: { profile: apikeyProfile } } = attributes;
+    const callerImsOrgId = apikeyProfile?.getImsOrgId() ?? requestContext.pathInfo.headers['x-gw-ims-org-id'];
```

Comment:
I agree that this should be temporary. The API Key profile should be the ony source of truth for the IMS Org Id. We shouldn't allow the IMS Org Id to be spoofed.

Reply:
I can add an IMS_ORG_ID to your keys.